### ISSUE TITLE
Hydration speed up

### ISSUE THREAD

**Issue by lastonoga** [sveltejs/sapper#1305]
Hey, guys!

I did a work to speed up hydration step (https://github.com/sexy-framework/sexy).
And i got really good numbers (better then NuxtJS in 4.5 times and Sapper in 10 times)

And i can try to help you with it.

To make really cool FID and TTI we need to skip first hydration.

First "DOM hydration" is made by SSR, then on client side **we only** need to register subscribers on reactive variables and skip DOM work:
- innerHTML
- nodeValue
- setAttribute
- classList and etc

All of that we need to skip.

As i know Svelte doesn't skip it. Am i right?

**Comment by benmccann** [comment]
Yes, I agree 100% and suggested the same here: https://github.com/sveltejs/svelte/issues/4975. There's also https://github.com/sveltejs/svelte/issues/4308, which has a PR, but I think still does too much work

In any case, I think this issue should probably be closed because it's an issue in Svelte (not Sapper) and there are already existing issues tracking it

**Comment by lastonoga** [comment]
@benmccann Looks like svelte maintainers don't want to make fast hydration...

**Comment by slimboyfats** [comment]
@lastonoga Why do you think that?

**Comment by lastonoga** [comment]
@slimboyfats First, because PR was ready 5 months ago. And second, bad performance makes me to not use Svelte even if i like it.

Thats why i made my own framework with super-fast hydration. I don't want to create bikes but what else can i do? Vue/React/Angular are very slow.

**Comment by slimboyfats** [comment]
@lastonoga But the PR isn't closed so I don't find where it's denied, can you link?

**Comment by lastonoga** [comment]
@slimboyfats It's not denied. Simple performance question is under discussion for 5 months and thats a problem for framework progress. Vue will make  new version in 6 months and it will be faster then Svelte for sure. And svelte will loose market. I did framework that has 10x faster hydration then Svelte, but has same features. It doesn't look like a problem to accept a PR.

**Comment by slimboyfats** [comment]
But I read in the issue that it might be breaking changes and that is a major thing to do.

**Comment by antony** [comment]
@lastonoga It's not a race to beat Vue, and Svelte doesn't have bad performance. this Issue is raised on the wrong project, and duplicates existing issues. Furthermore, we don't just shove in every performance enhancement, as, has been made evident in the last few days, most of them have huge glaring issues which would regress various features of the framework.

We will happily accept properly considered PRs which are well written, tested, and consider every aspect of the impact of the code changing including breaking existing users of the codebase.

One of the best bits about Svelte is it's simplicity, and the care taken to provide the optimal DX, instead of hammering it for performance, breaking various existing scenarios, and shifting CPU cycles to the browser, when it's already significantly faster than most other things out there.

If you can't raise issues in a neutral manner without a pressure-driven doomsday scenario because we haven't acknowledged and merged your framework into ours within 24 hours, then it might be better to not raise them at all.
